src/clarity-angular/wizard/_variables.wizard.scss

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+$clr-wizard-main-bgcolor: clr-getColor(1);
```

Comment:
These default variables don't have `!default` keyword in their values so users wouldn't be able to overwrite them if they want to build on their own.

Reply:
Most of the variables were migrated `as is` from the original location.
I do override several of the wizard colors for dark theme so this won't prevent users from overwriting them in a custom theme.